pom.xml

```diff
@@ -44,13 +44,18 @@
     <dependency>
       <groupId>com.moesif.api</groupId>
       <artifactId>moesifapi</artifactId>
-      <version>1.6.18</version>
+      <version>1.8.0</version>
```

Comment:
@bakennedy this needs update to `1.8.1` ?

Reply:
yeah that's a good idea.  This was on my review todo order first, so I didn't make that version update at the time, but might as well push this to latest